amy/templates/emails/scheduled_email_edit.html

```diff
@@ -15,4 +16,17 @@ <h1>
 
 {% crispy form %}
 
+<div class="row">
+  <div class="col-12 col-lg-2">
+    <strong>Template variables</strong>
+  </div>
+  <div class="col-12 col-lg-10">
+    <ul>
+    {% for key, value in body_context_annotations.items %}
```

Comment:
Can these template variables still be used when editing a scheduled email? I had assumed the templates were rendered once and weren't re-rendered before sending.
If they can't be used, this row should be removed from here and `scheduled_email_detail.html`.

Reply:
@elichad You're correct.

There are 2 renderings:
1. raw user input --(django template enigne)-> Markdown format
2. Markdown format --(markdown compiler)-> HTML

Currently first rendering occurs when an email is scheduled, and the second should be performed by the worker (at the moment it isn't - I will get back to this).

The advantage of this approach is simplicity, but the clear disadvantage is the final form of the email which can't be updated with newer versions of the objects included. So potentially we need to shift the rendering more towards the worker - it would have to render Django template and then Markdown.

I'll remove these lines, however they may be brought back in future.